src/app/CompilerPane.py

```diff
@@ -155,7 +155,12 @@
     ),
 ]
 
-vrad_light_type = tk.IntVar(value=COMPILE_CFG.get_bool('General', 'vrad_force_full'))
+# vrad_light_type = tk.IntVar(value=COMPILE_CFG.get_bool('General', 'vrad_force_full'))
+
+vrad_compile_type = tk.StringVar(
+    value=COMPILE_CFG.get_val('General', 'vrad_compile_type', 'FAST')
```

Comment:
Instead of hardcoding FAST here, it'd be good to fetch the old config and use that as the default, so we automatically transfer the previous configuration.